ng-dev/utils/git/github.ts

```diff
@@ -31,7 +31,7 @@ export interface GithubRepo {
 /** A Github client for interacting with the Github APIs. */
 export class GithubClient {
   /** The octokit instance actually performing API requests. */
-  private _octokit = new Octokit(this._octokitOptions);
+  protected _octokit = new Octokit({...this._octokitOptions, request: {fetch}});
```

Comment:
Any reason for this?

Reply:
Starting with the new major of Octokit, you have to provide fetch, or it will attempt to fall back to the native fetch, but this is only available in node 18+.

Reply:
Can you add a TODO for node 18 to remove this then?